adapters/Fyber/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Fyber iOS Mediation Adapter Changelog
 
+#### [Version 8.0.0.0](https://dl.google.com/googleadmobadssdk/mediation/ios/fyber/FyberAdapter-8.0.0.0.zip)
```

Comment:
(In progress)     Could you rename it since the URL is not yet live.

Reply:
sure